test/append.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var eq = require('./utils').eq;
+var S = require('..');
+
+describe('append', function() {
```

Comment:
I believe the convention held within this project is to add two newlines after the last import statement.